agent/cert-monitor/cert_monitor_test.go

```diff
@@ -557,6 +585,13 @@ func TestCertMonitor_Fallback(t *testing.T) {
 		VerifyServerHostname: true,
 	}, nil).Once()
 
+	cm.persist.On("persist", &structs.SignedResponse{
+		IssuedCert:           *secondCert,
+		ConnectCARoots:       *cm.initialRoots,
+		ManualCARoots:        cm.extraCACerts,
+		VerifyServerHostname: cm.verifyServerHostname,
+	}).Return(nil).Once()
```

Comment:
Looking at this made me realize that we should also call the `persist` callback with the response from the `fallback` callback so that we properly persist those certificates as well.

Reply:
Done